internal/server/placepage/place_page.go

```diff
@@ -215,13 +215,21 @@ func fetchBtData(
 	store *store.Store,
 	places []string,
 	statVars []string,
+	category string,
 ) (map[string]*pb.StatVarSeries, map[string]*pb.PointStat, error) {
 	// Fetch place page cache data in parallel.
+	action := [][]string{places}
+	prefix := bigtable.BtPlacePagePrefix
+	if category != "" {
+		action = [][]string{places, {category}}
+		prefix = bigtable.BtPlacePageCategoricalPrefix
+	}
```

Comment:
Generally, for PRs where we're in the process of migrating caches/data, its good to point out in the Description as to why the change is safe.

The change is a NOP until website starts setting `category`.  And before we do that, we should ensure all import-groups are rebuilt with the new sharded cache.